ui_src/src/domain/schema/components/createSchema/index.js

```diff
@@ -40,6 +40,7 @@ import draft7MetaSchema from 'ajv/dist/refs/json-schema-draft-07.json';
 import Ajv2020 from 'ajv/dist/2020';
 import draft6MetaSchema from 'ajv/dist/refs/json-schema-draft-06.json';
 import GenerateSchema from 'generate-schema';
+import { parse } from 'graphql';
```

Comment:
it seems you are using a package called graphql although I can't find it in the package json

Reply:
removed unnecessary packages and added necessary ones in package.json